.gitignore

```diff
@@ -0,0 +1,2 @@
+/*build*
+*.user
```

Comment:
Is this required? And if it is, is it enough? I didn't build this project individually and don't know where intermediates get put, but if MSVC scatters them around, other build systems might as well?

Reply:
I don't get what the problem is, this is a file that ignores files that should not be part of the git tree, if other IDEs adds other files they can be added in a later time.

Reply:
It was just a question. I just wondered if you start to ignore intermediates from one build system, wouldn't it be a more complete solution to ignore intermediates from the other build systems in use.